qual_id/pattern.py

```diff
@@ -19,6 +20,7 @@ class Pattern:
       "game":Game(),
       "geography":Geography(),
       "instrument": Instrument()
```

Comment:
```suggestion
      "instrument": Instrument(),
```
This is making the tests fail, I'll fix it when I resolve the conflicts. They're just due to other PRs getting merged 🙂